ProcessLib/HeatTransportBHE/BHE/BHE_1U.h

```diff
@@ -45,9 +45,10 @@ class BHE_1U final : public BHECommonUType
            RefrigerantProperties const& refrigerant,
            GroutParameters const& grout,
            FlowAndTemperatureControl const& flowAndTemperatureControl,
-           PipeConfigurationUType const& pipes)
+           PipeConfigurationUType const& pipes,
+		   bool const usePythonBC)
         : BHECommonUType{borehole, refrigerant, grout,
-                         flowAndTemperatureControl, pipes}
+                         flowAndTemperatureControl, pipes,usePythonBC}
```

Comment:
`git clang-format`

Reply:
clang-format performed on all related files. Changes included in [d3d3e3](https://github.com/ufz/ogs/pull/2702/commits/d3d3e3ab5a70ba7be9fd3c632e729f71c0e922c1)